src/core/navigation/appBar.component.tsx

```diff
@@ -15,6 +15,14 @@ import {
   TopNavigationBarActionProps,
 } from '@kitten/ui';
 
+interface NavigationState {
+  index: number;
+  routes: any[];
```

Comment:
type is NavigationRoute

Reply:
also this type is declared in react-navigation I think

Reply:
Will be refactored in the next PR.